test/e2e/workspacetype/controller_test.go

```diff
@@ -156,34 +163,37 @@ func TestClusterWorkspaceTypes(t *testing.T) {
 				kubeClusterClient, err := kcpkubernetesclientset.NewForConfig(cfg)
 				require.NoError(t, err, "failed to construct kube cluster client for server")
 
-				framework.AdmitWorkspaceAccess(t, ctx, kubeClusterClient, universal, []string{"user-1"}, nil, []string{"access"})
+				framework.AdmitNewWorkspaceAccess(t, ctx, kubeClusterClient, universal, []string{"user-1"}, nil, []string{"access"})
 				cr, crb := createClusterRoleAndBindings(
-					"create-clusterworkspace-user-1",
+					"create-workspace-user-1",
 					"user-1", "User",
-					tenancyv1alpha1.SchemeGroupVersion.Group,
-					"clusterworkspaces",
+					tenancyv1beta1.SchemeGroupVersion.Group,
+					"workspaces",
 					"",
 					[]string{"create"},
 				)
-				t.Logf("Admit create clusterworkspaces to user-1 in universal workspace %q", universal)
+				t.Logf("Admit create workspaces to user-1 in universal workspace %q", universal)
 				_, err = kubeClusterClient.Cluster(universal).RbacV1().ClusterRoles().Create(ctx, cr, metav1.CreateOptions{})
 				require.NoError(t, err)
 				_, err = kubeClusterClient.Cluster(universal).RbacV1().ClusterRoleBindings().Create(ctx, crb, metav1.CreateOptions{})
 				require.NoError(t, err)
 
-				cr, crb = createClusterRoleAndBindings(
-					"apiexport-user-1-"+universal.String(),
-					kcpapisv1alpha1.MaximalPermissionPolicyRBACUserGroupPrefix+"user-1", "User",
-					tenancyv1alpha1.SchemeGroupVersion.Group,
-					"clusterworkspaces",
-					"",
-					[]string{"create"},
-				)
-				t.Logf("Admit create clusterworkspaces for api bindings to %q in root workspace %q", kcpapisv1alpha1.MaximalPermissionPolicyRBACUserGroupPrefix+"user-1", tenancyv1alpha1.RootCluster)
-				_, err = kubeClusterClient.Cluster(tenancyv1alpha1.RootCluster).RbacV1().ClusterRoles().Create(ctx, cr, metav1.CreateOptions{})
-				require.NoError(t, err)
-				_, err = kubeClusterClient.Cluster(tenancyv1alpha1.RootCluster).RbacV1().ClusterRoleBindings().Create(ctx, crb, metav1.CreateOptions{})
-				require.NoError(t, err)
+				// TODO (p0lyn0mila): check if it is safe to remove the following RBAC
+				/*
+					cr, crb = createClusterRoleAndBindings(
+						"apiexport-user-1-"+universal.String(),
+						kcpapisv1alpha1.MaximalPermissionPolicyRBACUserGroupPrefix+"user-1", "User",
+						tenancyv1alpha1.SchemeGroupVersion.Group,
+						"clusterworkspaces",
+						"",
+						[]string{"create"},
+					)
+					t.Logf("Admit create clusterworkspaces for api bindings to %q in root workspace %q", kcpapisv1alpha1.MaximalPermissionPolicyRBACUserGroupPrefix+"user-1", tenancyv1alpha1.RootCluster)
+					_, err = kubeClusterClient.Cluster(tenancyv1alpha1.RootCluster).RbacV1().ClusterRoles().Create(ctx, cr, metav1.CreateOptions{})
+					require.NoError(t, err)
+					_, err = kubeClusterClient.Cluster(tenancyv1alpha1.RootCluster).RbacV1().ClusterRoleBindings().Create(ctx, crb, metav1.CreateOptions{})
+					require.NoError(t, err)
+				*/
```

Comment:
This can be deleted. Workspaces don't have the old restriction anymore.

Reply:
done